### DISCUSSION TITLE
How to Install ffmpeg

### DISCUSSION THREAD

**Discussion by rasazu** [MiguelRipoll23/homebridge-securitysystem#150] [Q&A] [answered]
Hi,
It's me again.....
How can I install ffmpeg for homebridge-securitysytsem? Can you provide some more detailed instructions.

I'm running Homebridge in Raspbian GNU/Linux Buster (10)
Versión de Node.js:  v14.15.5
Version npm: v6.14.11

Thanks !!!

**Comment by MiguelRipoll23** [comment, marked as answer]
Just execute the following line:

```
sudo apt install ffmpeg
````
